tests/sentry/api/endpoints/test_organization_metric_data.py

```diff
@@ -93,7 +95,7 @@ def test_groupby_session_status(self):
                 self.store_session(
                     self.build_session(
                         project_id=self.project.id,
-                        started=(time.time() // 60 - minute) * 60,
+                        started=(self.now() - timedelta(minutes=minute)).timestamp(),
```

Comment:
This is not equivalent to previous test case 
The previous test case was applying a `(time.time() // 60 - minute) * 60` which would be equivalent to `self.now().replace(seconds=0) - timedelta(minutes=minute)`

Reply:
Oh right, that makes sense